docs/tutorial/index.rst

```diff
@@ -282,6 +285,270 @@ you will access the build logs,
+   :emphasize-lines: 3-5
+
+   python:
+     version: "3.9"
+     install:
+       - method: pip
+         path: .
+
+With this change, Read the Docs will install the Python code in the virtual environment
+before starting the Sphinx build, which will finish seamlessly.
+If you go now to the API page of your HTML documentation,
+you will see the ``lumache`` summary!
+
+Enabling PDF and EPUB builds
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sphinx can build several other formats in addition to HTML, such as PDF and EPUB.
+You might want to enable this formats for your project
+so your users can read the documentation offline.
+
+To do so, add this extra content to your ``.readthedocs.yaml``:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+   :emphasize-lines: 4-6
+
+   sphinx:
+     fail_on_warning: true
+
+   formats:
+     - pdf
+     - epub
+
+After this change, PDF and EPUB downloads will be available
+both from the "Downloads" section of the :term:`project home`,
+as well as the :term:`flyout menu`.
+
+.. figure:: /_static/images/tutorial/flyout-downloads.png
+   :align: center
+   :alt: Downloads available from the flyout menu
+
+   Downloads available from the flyout menu
+
+Versioning documentation
+------------------------
+
+.. Explain how to manage versions on RTD: create release branches,
+   activate the corresponding version, browse them in the version selector,
+   selectively build versions
+
+Read the Docs allows you to have :doc:`several versions of your documentation </versions>`,
+in the same way that you have several versions of your code.
+By default, it creates a ``latest`` version
+that points to the default branch of your version control system
+(``main`` in the case of this tutorial),
+and that's why the URLs of your HTML documentation contain the string ``/latest/``.
+
+Creating a new version
+~~~~~~~~~~~~~~~~~~~~~~
+
+Let's say you want to create a ``1.0`` version of your code,
+with a corresponding ``1.0`` version of the documentation.
+For that, first navigate to your GitHub repository, click on the branch selector,
+type :kbd:`1.0.x`, and click on "Create branch: 1.0.x from 'main'"
+(more information `on their documentation`__).
+
+__  https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository
+
+Next, go to your :term:`project home`, click on the "Versions" button,
+and under "Active Versions" you will see two entries:
+
+- The ``latest`` version, pointing to the ``main`` branch, and "Public".
+- A new ``stable`` version, pointing to the ``origin/1.0.x`` branch, and also "Public".
+
+.. figure:: /_static/images/tutorial/active-versions.png
+   :width: 80%
+   :align: center
+   :alt: List of active versions of the project
+
+   List of active versions of the project
+
+Right after you created your branch,
+Read the Docs created a new special version called ``stable`` pointing to it,
```

Comment:
maybe mention that stable is automatically updated to point to the latest release/tag, if your project follows semver.

Reply:
I'm not very happy with our docs at the moment (#8476) and saying here "because it follows semver" it's a lie, because `1.0.x` does _not_ follow semver. I am rephrasing this part.

Reply:
I don't believe `1.0.x` will work for `stable`.